.github/workflows/repo--validate-pr-title.yml

```diff
@@ -41,6 +41,7 @@ jobs:
             protocol
             relayer
             taikoon
+            ui-lib
```

Comment:
Let's move the ui lib setup to a separate PR that gets merged before the actual ui-lib package. Otherwise it will show up in the taikoon changelogs.

Reply:
https://github.com/taikoxyz/taiko-mono/pull/17219